packages/maskbook/src/plugins/RedPacket/UI/RedPacketInList.tsx

```diff
@@ -25,6 +37,60 @@ const useStyles = makeStyles((theme: Theme) =>
             fontSize: 12,
             color: theme.palette.text.secondary,
         },
+        root: {
+            borderRadius: 10,
+            border: `solid 1px ${theme.palette.divider}`,
+            marginBottom: theme.spacing(1.5),
+            position: 'static !important' as any,
```

Comment:
🤔

Reply:
Type '"static !important"' is not assignable to type 'Position | PropsFunc<{}, Position | undefined> | undefined'.

Reply:
hmm but why u need !important